src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const populationSpans = document.querySelectorAll('span.population');
+const totalPopSpan = document.querySelector('span.total-population');
+const averagePopSpan = document.querySelector('span.average-population');
+
+const totalPopulation = [ ...populationSpans ]
+  .map(span => Number(span.innerHTML.split(',').join('')) || 0)
+  .reduce((sum, population) => sum + population);
+
+const averagelPopulation = totalPopulation / populationSpans.length;
+
+totalPopSpan.innerHTML = totalPopulation.toLocaleString();
+averagePopSpan.innerHTML = averagelPopulation.toLocaleString();
```

Comment:
I am not sure about correctly use property for totalPopSpan & averagePopSpan

Reply:
you are probably right, innerText seems like a better property to use